content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -471,6 +472,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
    [Enabling Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpointslices/).
 - `GCERegionalPersistentDisk`: Enable the regional PD feature on GCE.
 - `GenericEphemeralVolume`: Enables ephemeral, inline volumes that support all features of normal volumes (can be provided by third-party storage vendors, storage capacity tracking, restore from snapshot, etc.). See [Ephemeral Volumes](/docs/concepts/storage/ephemeral-volumes/).
+- `GracefulNodeShutdown`: Enables support for graceful shutdown in kubelet. During a system shutdown, kubelet will attempt to detect the shutdown event and terminate pods running on the node with a specified `terminationGracePeriodSeconds`. See [KEP 2000 Graceful Node Shutdown](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2000-graceful-node-shutdown/README.md) for more details.
```

Comment:
> See [KEP 2000 Graceful Node Shutdown](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2000-graceful-node-shutdown/README.md) for more details.

We avoid linking to developer docs (such as KEPs) for documenting cluster behaviour. It's better to add a paragraph into https://kubernetes.io/docs/concepts/architecture/nodes/ and hyperlink to that fragment.

Reply:
Thanks, I added some more info on that page as you suggested. PTAL again.